content/applications/sales/sales/amazon_connector/manage.rst

```diff
@@ -53,6 +53,29 @@ turn, notify the customer that the order (or a part of it) is on its way.
 .. seealso::
    - :doc:`../../../inventory_and_mrp/inventory/shipping/setup/third_party_shipper`
 
+.. _manage/manage_delivery_errors:
+
+Manage errors when synchronizing the deliveries
+-----------------------------------------------
+
+When Odoo verifies that all the information sent to Amazon are correctly processed, it can happen
+that Amazon failed to do so. In that case, Odoo send an email listing all the shipments that failed
+and the errors Amazon sent with it. In addition, these shipment will be flagged with a
+:guilabel:`Synchronization with Amazon failed` tag.
+
+Usually, the error will be clear enough for you to correct the problem, so you either manage this
+order directly in Amazon backend, or you correct the problem in Odoo and synchronize the shipment
```

Comment:
Editing for clarity and wordiness. We also want to avoid using second-person language ("you") in order to make the writing sound more neutral and direct.
```suggestion
Usually, the error can be corrected directly in the Amazon backend or in Odoo. If the problem is corrected in Odoo, synchronize the shipment
```